src/IIIFPresentation/API.Tests/Integration/GetManifestTests.cs

```diff
@@ -269,7 +267,7 @@ public async Task Get_IiifManifest_Flat_ReturnsManifestFromFinalS3_IfStagingMiss
     public async Task Get_IiifManifest_Flat_ReturnsManifestFromS3_DecoratedWithPaintedResources()
     {
         // Arrange - add manifest with 1 canvasPainting with an asset and corresponding manifest in S3
-        var id = nameof(Get_IiifManifest_Flat_ReturnsManifestFromS3_DecoratedWithPaintedResources);
+        var id = $"{nameof(Get_IiifManifest_Flat_ReturnsManifestFromS3_DecoratedWithPaintedResources)}_returnsPainted";
```

Comment:
Small, worth adding a helper to `TestIdentifiers` so that it uses `[CallerMemberName]` and avoids potential c+p issues?